packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -237,19 +237,34 @@ async function getOutlookVersion(): Promise<string | undefined> {
   }
 }
 
-async function getOutlookExePath(): Promise<string> {
+async function getWXPOExePath(app: OfficeApp): Promise<string> {
+  let HostApp: string = "";
   try {
-    const OutlookInstallPathRegistryKey: string = `HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\OUTLOOK.EXE`;
-    const key = new registry.RegistryKey(`${OutlookInstallPathRegistryKey}`);
-    const outlookExePath: string | undefined = await registry.getStringValue(key, "");
-
-    if (!outlookExePath) {
-      throw new Error("Outlook.exe registry empty");
+    switch (app) {
+      case OfficeApp.Excel:
+        HostApp = "EXCEL";
```

Comment:
Assuming the registry look up is case sensitive I don't believe this (or the others that are not outlook) are correct.

Reply:
Yes, I will double check it in registry Editor. Thanks Darren!